packages/block-editor/src/components/block-patterns-list/index.js

```diff
@@ -77,6 +98,7 @@ function BlockPatternList( {
 	onClickPattern,
 	orientation,
 	label = __( 'Block Patterns' ),
+	isPatternTitleTooltip,
```

Comment:
The name of this prop is a bit weird, maybe just showTooltip or something?

Reply:
😅 `showTooltip` was my initial name and thought it drills down a few components and could be ambiguous. I can rename.

Reply:
could be something like `showTitlesAsTooltip` maybe as well